src/static/json/PhysicsRoutines/DoseFromFluence.json

```diff
@@ -33,8 +33,8 @@
       "type": "input",
       "parameterName": "fluence_cm2",
       "label": "Fluence [1/cm2]",
-      "placeholder": "100000000",
-      "defaultValue": 100000000,
+      "placeholder": "1000000000",
+      "defaultValue": 1000000000,
```

Comment:
1000 could be replaced with 1e3, 0.01 as 1e-2 etc.

I did some tests and both solutions works fine.